pages/linux/flatpak-builder.md

```diff
@@ -0,0 +1,24 @@
+# flatpak-builder
+
+> Help build application dependencies.
+> More information: <https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html>.
+
+- Build a Flatpak then export it to a new repository:
+
+`flatpak-builder {{path/to/build_directory}} {{path/to/manifest}}`
+
+- Build a Flatpak then export it to the specified repository:
+
+`flatpak-builder --repo={{repository_name}} {{path/to/build_directory}} {{path/to/manifest}}`
+
+- Build a Flatpak then install it locally:
+
+`flatpak-builder --install {{path/to/build_directory}} {{path/to/manifest}}`
+
+- Build and sign a Flatpak then export it to the specified repository:
+
+`flatpak-builder --gpg-sign={{key_id}} --repo={{repository_name}} {{path/to/manifest}}`
```

Comment:
Why not just `key`?

Reply:
I was afraid someone might actually try to paste the full private/public key. 😆 
I thought this might be more friendly for people new to GPG.

If you disagree with my logic, I'll update it.

Reply:
I don't really know much about GPG, I just thought that in most cases a key already is an ID. So, if it makes sense we should keep it.

Reply:
Nah, a key is a giant mess.

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBGCmjRcBEAC234uq2uEzRHggxJYVngegBiyYjQpSjVVbmtKAsCxq898VapIg
zg1nJh33WFJlrsJctEmFdpT/r9KARi1Bup7f8ZSkjVSwyZW1IAXqSmootE92nt8Z
YmFNcvBHVcTaRsJk3myLztSA928zG2cJSln8uhVbjTig6PFrFWGYX5MBjYvrT80u
eLHYf/GConJm+sUhqRlYrUspINEGxDvOoFYddyy2tQ12FBY+8KGMOsK4M2YQVAd3
...
```

While the ID is just a small string like:
* fingerprint - `94FC18B28A1505EF7E3DE3EADE1C217EFF01FEC8`
* 64-bit ID - `DE1C217EFF01FEC8`
* 32-bit ID - `FF01FEC8`

Reply:
`key_id` is fine with me.